release-schema.json

```diff
@@ -50,6 +62,38 @@
             "null"
           ],
           "minLength": 1
+        },
+        "address": {
+          "title": "Address",
+          "description": "The address of the person.",
+          "$ref": "#/definitions/Address"
+        },
+        "email": {
+          "title": "Email",
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "The e-mail address of the person.",
```

Comment:
Move `description` (here and elsewhere) under `title`.

Reply:
Done in https://github.com/open-contracting-extensions/ocds_beneficialOwners_extension/pull/2/commits/3ee0f372ffe4c3facdd4b819edfe15be46e1a66a